Vagrantfile

```diff
@@ -56,7 +56,7 @@ Vagrant.configure('2') do |config|
   config.vm.post_up_message = <<~EOT
     Log into the Vagrant box with \`vagrant ssh\`
       Run the test suite by \`./bin/rake\`
-      Start Rails server by \`./bin/rails server\`
+      Start Rails server by \`shoreman start\`
```

Comment:
Not used shoreman before, but I think this cmd is wrong: `/usr/local/bin/shoreman: line 84: start: No such file or directory`.

Should it just be `shoreman`?